Engines/Wine/Engine/Implementation/script.js

```diff
@@ -69,8 +72,8 @@ var engineImplementation = {
                 wizard.close();
             }
         }
-    },
-    _installWinePackage: function (setupWizard, winePackage, localDirectory) {
+    }
+    _installWinePackage(setupWizard, winePackage, localDirectory) {
```

Comment:
Do you want to keep the `_` notation here?

Reply:
No. I didn't change the notation yet because the class is quite long and I only wanted to get it working for now. Long-term I think we should change this